src/mainwindow.cpp

```diff
@@ -6618,13 +6618,22 @@ void MainWindow::noteTextEditCustomContextMenuRequested(
     menu->addAction(ui->actionAutocomplete);
     menu->addAction(ui->actionSplit_note_at_cursor_position);
 
+
+
+    // add the scripts menu
+    QIcon scriptIcon = QIcon::fromTheme(
+        QStringLiteral("story-editor"),
+        QIcon(QStringLiteral(
+            ":icons/breeze-qownnotes/16x16/story-editor.svg")));
+    QMenu *scriptMenu = menu->addMenu(tr("Scripts"));
```

Comment:
Is this menu also added if there are no scripts? I guess we don't want that.

Reply:
I changed it.